example.ipynb

```diff
@@ -127,9 +127,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (jupyterlab2-h5web)",
+   "display_name": "Python 3",
    "language": "python",
-   "name": "jupyterlab2-h5web"
+   "name": "python3"
```

Comment:
Putting standard kernel name: I always had errors when opening the notebook elsewhere